netshare/generate/model_generate.py

```diff
@@ -1,89 +1,117 @@
 import copy
-import subprocess
-import sys
-import time
 import os
-import json
-import importlib
-import random
-import pandas as pd
-import socket
-import struct
-import ipaddress
-import argparse
+import time
+from pathlib import Path
+from typing import Dict, List
 
 import numpy as np
 import pandas as pd
-
-import netshare.ray as ray
-from pathlib import Path
 from tqdm import tqdm
-from scapy.all import IP, ICMP, TCP, UDP
-from scapy.all import wrpcap
-from scipy.stats import rankdata
-from pathlib import Path
+
+import netshare.utils.ray as ray
+from netshare import models
+from netshare.utils.logger import TqdmToLogger, logger
+from netshare.utils.model_configuration import create_chunks_configurations
+from netshare.utils.paths import get_generated_data_log_folder
+
+
+def model_generate() -> None:
+    configs, config_group_list = create_chunks_configurations(generation_flag=True)
+
+    logger.info("Start generating attributes")
+    ray.get(
+        [
+            generate_data.remote(
+                config=config,
+                given_data_attribute_flag=False,
+            )
+            for config in configs
+        ]
+    )
+    time.sleep(10)
+
+    # TODO: The below part didn't happen in dg. Why?
```

Comment:
DG considers a single chunk so we do not need to do this. This technique was introduced in NetShare paper. In general, if the user specifies to use a single chunk, we may just use the DG processing flow and many steps (including this) can be skipped.